lib/core/src/model.rs

```diff
@@ -1731,6 +1783,27 @@ impl Utxo {
     }
 }
 
+/// An argument when calling [crate::sdk::LiquidSdk::fetch_payment_proposed_fees].
+#[derive(Debug, Clone)]
+pub struct FetchPaymentProposedFeesRequest {
+    pub swap_id: String,
+}
+
+/// Returned when calling [crate::sdk::LiquidSdk::fetch_payment_proposed_fees].
+#[derive(Debug, Clone, Serialize)]
+pub struct FetchPaymentProposedFeesResponse {
+    pub swap_id: String,
+    pub fees_sat: u64,
+    /// Amount sent by the swap payer
+    pub payer_amount_sat: u64,
```

Comment:
Do you think its worth adding the amount that would be received also (without the dev having to calculate it)?

Reply:
Why not... Added in [fa70c0b](https://github.com/breez/breez-sdk-liquid/pull/615/commits/fa70c0b6dfc35cedaccbee745b9fd7100f8997a9)